profile/README.md

```diff
@@ -0,0 +1,31 @@
+## Hi there 👋
+
+🙋‍♀️ OpenSCD product vision
+
+For utility/integrator/vendor companies​
+
+Who want to have flexibility, agility and efficiency in their SCL configurations​
+
+**OpenSCD**​
+
+is an IEC 61850 SCL configuration tool​
```

Comment:
Configuration is not completely correct. We are aiming to become one, but are not there yet. That is why in the open-scd repository it says it is and SCL editor.

Reply:
how about: "OpenSCD aims to be an IEC 61850 SCL configuration tool"

Reply:
I still like the word `SCL editor` as this is describing it well and does not have any stigma to it. I however, also feel that it could be a bit more beneficial for potential other collaborators to name-drop some buzzwords maybe like so: `OpenSCD aims to provide modules to do system configuration as well as system specification`